docs-rtd/howto/manage-charms.md

```diff
@@ -24,13 +20,31 @@ resource "juju_application" "this" {
 }
 ```
 
+You can also specify a charm channel and revision. For example:
+
+
+```terraform
+resource "juju_application" "this" {
+    model = <model>
+    charm {
+        name     = "<charm-name>"
+        channel  = "<channel-name>"
+        revision = "<revision-number>"
+    }
+}
+```
+
+- If both `channel` and `revision` are specified (recommended for reproducibility), the Terraform provider will deploy the requested revision. (The channel will be used later when you update the charm, though see [issue 498](https://github.com/juju/terraform-provider-juju/issues/498) and [issue 881](https://github.com/juju/terraform-provider-juju/issues/881).)
+- If only `channel` is specified, the provider will deploy the latest revision available in that channel at application creation time. The charm will not be refreshed on subsequent `terraform apply` runs.
+- If only `revision` is specified, the provider will deploy that revision from the `stable` channel.
+- If neither field is specified, the provider will deploy the latest revision from the `latest/stable` channel at application creation time. The charm will not be refreshed on subsequent `terraform apply` runs.
```

Comment:
It will deploy the latest revision from the default channel, which may not be `latest/stable`.